### ISSUE TITLE
Stash the active (or selected) tabs option

### ISSUE THREAD

**Issue by redactedscribe** [josh-berry/tab-stash#132]
### Problem Statement
I'd like to avoid the circumstances of potentially stashing all tabs. I'm more comfortable stashing a single tab at a time.

### Preferred Solution(s)
The options that the toolbar button / <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>W</kbd> is limited to are:
- Stash all (or selected)
- Stash active
- Do nothing

As indicated by the title of the issue, an intermediary option of "Stash the active _(or selected)_ tabs" would be appreciated.

This would make the toolbar button / keyboard shortcut more useful to me: Stashes a single tab at a time (no large mistakes), or stash a bunch of selected tabs at once (unlikely to be a mistake due to the explicit steps required).

### Alternative Solution(s)
Maybe "Stash active" option should default to being "(or selected)" just like the first option? No need for a fourth option then, but rather a modification to the current option.

<hr>
<em>Readers: If you are also interested in seeing this feature be developed, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the feature request, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
Yeah, I am thinking of revisiting this in a major (like 3.0) release; see #74, because I also find that most of the time, I myself do not want to stash ALL tabs, just some subset. The blocker here is having true multi-selection in the UI (#57), so that it's still easy/convenient to stash lots of tabs.

Do you think #74 would cover the issue you're describing here? Or is there still something I'm missing that isn't covered by that issue?

**Comment by redactedscribe** [comment]
Thanks for the reply. 

#74 should cover my use case I believe; Stash Tab / Stash Selected Tabs as a combined action, and Stash All as its own action. I dislike the stash all feature enough that I'd prefer it not even being displayed in any of the context menus. Every time I want to stash tabs, I become acutely aware of wanting to avoid hitting "Stash Tabs" menu option by mistake. Maybe it could be optionally tucked away somewhere so it's still accessible for the rare cases I'd want it, but to not be a listed option using my preferred method of accessing the actions. Maybe it's a bit irrational, but it's really quite annoying to accidentally trigger that feature, in OneTab too.

The only other thing I suggested was a separate shortcut dedicated to opening the UI so that you won't have to choose between doing that or stashing tabs.

**Comment by josh-berry** [comment]
Got it, thanks. I'll re-title this issue to cover just the keyboard shortcut then and keep #74 for the stash/stash selected issue.
